keyboards/ergodox_ez/keymaps/colemak_es_osx/config.h

```diff
@@ -0,0 +1,20 @@
+#include "../../config.h"
```

Comment:
This isn't actually necessary, as the build process includes all `config.h` files, automatically. 

However, this should be replaced with an include guard: 

```suggestion
#pragma once
```

Reply:
Fixed